constraint/pkg/client/crd_helpers.go

```diff
@@ -19,8 +19,10 @@ import (
 func validateTargets(templ *v1alpha1.ConstraintTemplate) error {
 	if len(templ.Spec.Targets) > 1 {
 		return errors.New("Multi-target templates are not currently supported")
+	} else if templ.Spec.Targets == nil {
```

Comment:
Shouldnt we check for nil before we check the length?

Reply:
Length is zero if the list is nil.